### DISCUSSION TITLE
Does the frontend support the pushing message such as WebSocket

### DISCUSSION THREAD

**Discussion by ZXueW** [frangoteam/FUXA#1502] [Q&A]
Hi, notice that the frontend retrieves the device values by polling as shown in this picture below.
![image](https://github.com/user-attachments/assets/7b20741a-762d-4a09-b465-f99afbe35c8c)
Can it support pushing message such as WebSocket? If not, can the polling interval be changed shorter? Thank you.

**Comment by MatthewReed303** [comment]
@ZXueW if you need to use WebSocket, I added it to node,js package list in latest docker image, you can now use WebSocket in a script using node.js, in my case I use it to communicate to Node-Red. I will add some details to the Wiki

**Comment by ZXueW** [comment]
@MatthewReed303 WebSocket is good enough, look forward to the Wiki update. Thank you.

**Reply by MatthewReed303** [reply]
@ZXueW please see the Wiki have just added websocket client example. https://github.com/frangoteam/FUXA/wiki/HowTo-WebSockets

**Reply by ZXueW** [reply]
@MatthewReed303, thank you, seems great.  Still need your help, would you please show me where I should put this client code? After that, suppose I can observe the timely updated value from the Frontend by binding the device tag to some Control.
